doc/SPO-MANUAL-MIGRATION-GUIDE.md

```diff
@@ -0,0 +1,617 @@
+echo "" >> $BACKUP_DIR/backup-summary.txt
+
+echo "Profiles backed up by type:" >> $BACKUP_DIR/backup-summary.txt
+echo "  SeccompProfiles:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/profiles -name "seccompprofiles-*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+echo "  SELinuxProfiles:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/profiles -name "selinuxprofiles-*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+echo "  RawSELinuxProfiles:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/profiles -name "rawselinuxprofiles-*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+echo "" >> $BACKUP_DIR/backup-summary.txt
+echo "Profile bindings backed up:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/bindings-recordings -name "*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+
+cat $BACKUP_DIR/backup-summary.txt
+```
+
+**Actual output:**
+```
+=== Backup Summary ===
+Total files backed up: 6
+
+SPOD configuration: Yes
+
+Profiles backed up:
+rawselinuxprofiles-openshift-security-profiles-errorlogger.json
+seccompprofiles-openshift-security-profiles-profile-block-all.json
+selinuxprofiles-openshift-security-profiles-errorlogger-selinuxd-test.json
+
+Profile bindings backed up:
+profilebindings-openshift-security-profiles-profile-binding-seccomp.json
+profilebindings-openshift-security-profiles-profile-binding-selinuxd.json
+```
+
+## Step 4: Delete SPO Objects in Order
+
+**IMPORTANT**: Delete profile bindings first to prevent the operator from automatically reapplying profiles to workloads. You also need to make sure there is no activeWorkload in the
+profileBinding objects before deleting it.
+
+### Delete Profile Bindings and activeworkloads
+```bash
+# List all profile bindings across all namespaces
+oc get profilebindings -A
+
+# Check all profilebinding activeworkload field
+oc get profilebindings -A -o json | jq -r '.items[] | "\(.metadata.namespace)/\(.metadata.name): \(.status.activeWorkloads)"'
+
+# Check and Delete all workload listed there above
+
+
+# Delete all profile bindings from all namespaces
+for ns in $(oc get profilebindings -A -o jsonpath='{range .items[*]}{.metadata.namespace}{"\n"}{end}' | sort -u); do
+  echo "Deleting profile bindings in namespace: $ns"
+  oc delete profilebindings --all -n $ns
+done
+```
+
+**Actual command and output:**
+```bash
+Deleting profile bindings in namespace: openshift-security-profiles
+profilebinding.security-profiles-operator.x-k8s.io "profile-binding-seccomp" deleted
+profilebinding.security-profiles-operator.x-k8s.io "profile-binding-selinuxd" deleted
+```
+
+### Update Workloads to Remove Profile References
+
+Next, update all workloads to remove references to SPO profiles. You must either recreate pods without these profiles, or update deployments, statefulset, and other workloads to remove the profile references from their pod templates.
+
+
+### Delete Security Profiles
```

Comment:
Before deleting the security profiles, we need to delete the active workload for the security profiles first.